Header header
string text
uint8 severity
float32 rate

uint8 nothing = 0
uint8 info = 1
uint8 warning = 2
uint8 error = 3
uint8 critical_error = 4
uint8 fatal_error = 5
